player/public/coreplugins/org.ekstep.ecmlrenderer-1.0/renderer/qspatch.js

```diff
@@ -390,5 +390,70 @@ var qspatch = {
             'image': data.image ? data.image : undefined,
             'audio': data.audio ? data.audio : undefined,
         })
+    },
+    //Make default font size for question options
+    setDefaultFontSize: function(data){
+      if(data.stage['org.ekstep.questionset']){
+        try {
+          var questionSetData = data.stage['org.ekstep.questionset'];
+          _.each(questionSetData['org.ekstep.question'],function(item,key){
+              var questionData = JSON.parse(item.data.__cdata);
+              questionData.question.text = qspatch.changeFontSize(questionData.question);
+              item.data.__cdata = JSON.stringify(questionData);
```

Comment:
Why do you need this line? You can update the question after changing the question & options font-size

Reply:
Question object reference was lost so updating question and option text separately